randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -252,6 +254,7 @@ Extra - room_id: [5]
   * L0 Hatch to Nocturnal Playground/Door to Nocturnal Shaft
   * Extra - door_idx: (14,)
   > Door to Nocturnal Access
+      # Without Breaking Blocks: https://youtu.be/o43xM3s5YdE
```

Comment:
can you add in your video description to **not** shoot the blocks?

Reply:
Added